.github/workflows/test-cron.yaml

```diff
@@ -466,7 +466,7 @@ jobs:
 
         '
     - name: Run Python tests
-      run: './pants test ::
+      run: './pants test :: -- --noskip
```

Comment:
Oh wait, better is using our dedicated CI config file we already have to set this value

Reply:
Ooooh good point!